content/blog/lintr-3-0-0/index.md

```diff
@@ -0,0 +1,229 @@
+Each tag also gets its own documentation page (e.g. `?readability_linters`) which describes the tag
+and lists all of the known associated linters. The tags are available in another database:
+`available_tags()`. These databases can be extended to include custom linters in your package;
+see `?available_linters`.
+
+Moreover, lintr's documentation is now available as a website thanks to
+Hadley Wickham's contribution to create a pkgdown website for the package:
+[lintr.r-lib.org](https://lintr.r-lib.org).
+
+## Google linters
+
+This release also features more than 30 new linters originally authored by Google developers.
+Google adheres mostly to the tidyverse style guide and uses lintr to improve the quality
+of its considerable internal R code base. These linters detect common issues with
+readability, consistency, and performance. Here are some examples:
+
+ - `any_is_na_linter()` detects the usage of `any(is.na(x))`; `anyNA(x)` is nearly always a better choice,
+   both for performance and for readability
+ - `expect_named_linter()` detects usage in [testthat](http://testthat.r-lib.org/) suites like
+   `expect_equal(names(x), c("a", "b", "c"))`; `testthat` also exports `expect_named()` which is
+   tailor made to make more readable tests like `expect_named(x, c("a", "b", "c"))`
+ - `vector_logic_linter()` detects usage of vector logic operators `|` and `&` in situations where
+   scalar logic applies, e.g. `if (x | y) { ... }` should be `if (x || y) { ... }`. The latter
+   is more efficient and less error-prone.
+ - `strings_as_factors_linter()` helps developers maintaining code that straddles the R 4.0.0 boundary,
+   where the default value of `stringsAsFactors`
+   [changed from `TRUE` to `FALSE`](https://developer.r-project.org/Blog/public/2020/02/16/stringsasfactors/),
+   by identifying usages of `data.frame()` that (1) have known string columns and (2) don't declare
+   a value for `stringsAsFactors`, and thus rely on the R version-dependent default.
+
+See the [NEWS](https://lintr.r-lib.org/news/index.html#google-linters-3-0-0) for the complete list.
+
+## Other improvements
+
+This is a big release -- almost 2 years in the making -- and there has been a plethora of smaller
+but nonetheless important changes to lintr. Please check the
+[NEWS](https://lintr.r-lib.org/news/index.html#lintr-300)
+for a complete enumeration of these. Here are a few more new linters as a highlight:
+
+ - `sprintf_linter()`: a new linter for detecting potentially problematic calls to `sprintf()` (e.g.
+   using too many or too few arguments as compared to the number of template fields)
+ - `package_hooks_linter()`: a new linter to check consistency of `.onLoad()` functions and
+   other namespace hooks, as required by `R CMD check`
+ - `namespace_linter()`: a new linter to check for common mistakes in `pkg::symbol` usage, e.g.
+   if `symbol` is not an exported object from `pkg`
+
+# What's next in lintr
+
```

Comment:
I think you could includes the contents of this section (sans headings) in the "Google linters" section, since they're naturally related.

Reply:
Done, PTAL